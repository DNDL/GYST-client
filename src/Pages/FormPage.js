import React from 'react';
import PropTypes from 'prop-types';
import AddHabit from '../components/Containers/AddHabit';
import Nav from '../components/Nav/Nav';
import Logo from '../components/Logo/Logo';

function FormPage(props) {
  const { history } = props;
  return (
    <>
      <Logo />
      <AddHabit props={history}/>
      <Nav />
    </>
  );
}

FormPage.propTypes = {
  history: PropTypes.object
};

export default FormPage;
