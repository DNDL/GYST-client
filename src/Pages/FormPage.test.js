import React from 'react';
import { shallow } from 'enzyme';
import FormPage from './FormPage';

describe('FormPage component', () => {
  it('renders FormPage', () => {
    const wrapper = shallow(<FormPage history={{}}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
