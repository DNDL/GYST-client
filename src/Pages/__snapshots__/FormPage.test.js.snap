// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormPage component renders FormPage 1`] = `
<Fragment>
  <Logo />
  <Connect(AddHabit)
    props={Object {}}
  />
  <Nav />
</Fragment>
`;
